services/device-registry-base/src/main/java/org/eclipse/hono/deviceregistry/service/device/AbstractRegistrationService.java

```diff
@@ -77,19 +78,19 @@ public final void setTenantInformationService(final TenantInformationService ten
      *
      * @param deviceKey The ID of the device to get registration data for.
      * @param span The active OpenTracing span for this operation. It is not to be closed in this method! An
-     *            implementation should log (error) events on this span and it may set tags and use this span as the
-     *            parent for any spans created in this method.
+     * implementation should log (error) events on this span and it may set tags and use this span as the
+     * parent for any spans created in this method.
      * @return A future indicating the outcome of the operation. The <em>status</em> will be
-     *            <ul>
-     *            <li><em>200 OK</em>, if a device with the given ID is registered for the tenant.<br>
-     *            The <em>payload</em> will contain a JSON object with the following properties:
-     *              <ul>
-     *              <li><em>device-id</em> - the device identifier</li>
-     *              <li><em>data</em> - the information registered for the device</li>
-     *              </ul>
-     *            </li>
-     *            <li><em>404 Not Found</em>, if no device with the given identifier is registered for the tenant.</li>
-     *            </ul>
+     * <ul>
+     * <li><em>200 OK</em>, if a device with the given ID is registered for the tenant.<br>
+     * The <em>payload</em> will contain a JSON object with the following properties:
+     *   <ul>
+     *   <li><em>device-id</em> - the device identifier</li>
+     *   <li><em>data</em> - the information registered for the device</li>
+     *   </ul>
+     * </li>
+     * <li><em>404 Not Found</em>, if no device with the given identifier is registered for the tenant.</li>
+     * </ul>
```

Comment:
When I perform a "Reformat code" in my IDE using the Eclipse Code Formatter with the "hono-code-style.xml" configuration, there is an indentation here - both for "@params" and "@return". So, IMHO, we should keep the indentation. This also applies to other javadoc changes in this PR.